src/main/java/racingcar/RacingCarGameRule.java

```diff
@@ -0,0 +1,28 @@
+package racingcar;
+
+import racingcar.model.Car;
+import racingcar.ui.ResultView;
+import java.util.List;
+
+public class RacingCarGameRule {
+    private static final int MOVE_CONDITION = 3;
+    private static RacingCarGameRule instance = new RacingCarGameRule();
+
+    private RacingCarGameRule() {
+
+    }
+
+    public static RacingCarGameRule getInstance() {
+        return instance;
+    }
+
+    public void moveCars(List<Car> cars) {
+        cars.stream().forEach(car -> car.move(isMovingForward(RandomManaager.getInstance().getRandomValue())));
```

Comment:
```suggestion
        cars.forEach(car -> car.move(isMovingForward(RandomManager.getInstance().getRandomValue())));
```
바로 forEach를 사용할 수 있어요! + (RandomManager 오타 😅)

Reply:
코드 한줄이 너무 길다보니까 읽기 어려운 상황이에요
한줄에 모든 것을 포함하려는 것 보다는 변수 또는 메소드 추출을 사용하여 작성해보는 것이 어떨까요?